homeassistant/components/terncy/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "terncy",
+  "name": "Terncy",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/terncy",
+  "requirements": ["terncy==0.3.5","websockets==8.1","zeroconf==0.28.7"],
+  "zeroconf": ["_websocket._tcp.local."],
```

Comment:
This zeroconf type is too generic. Please limit is more using a mac or name filtering.

For more information see: <https://developers.home-assistant.io/docs/creating_integration_manifest/#zeroconf>